phpBB/phpbb/files/filespec_storage.php

```diff
@@ -111,7 +111,7 @@ public function __construct(language $language, \bantu\IniGetWrapper\IniGetWrapp
 	 */
 	public function set_upload_ary($upload_ary)
 	{
-		if (!isset($upload_ary) || !sizeof($upload_ary))
+		if (!isset($upload_ary) || !count($upload_ary))
```

Comment:
Can actually be simplified to just `empty($upload_ary)`...

Reply:
You should have told this to @rxu in his PR, i am doing here what he did in filespec.php, because this was desynchronized and was causing tests to fail

Reply:
@Elsensee #4862

Reply:
The logic is simple. I was replacing ```sizeof()``` with ```empty()``` for non-countable objects only. For countable ones (arrays) it was straight ```sizeof()``` -> ```count()``` replacement.

Reply:
Why don't you use `empty()` for them too? I think it's faster than `count` or `sizeof` for some arrays?

Reply:
If `$array` is already defined, why use `if (empty($array))`? Why not `if (!$array)`?

Reply:
>Why don't you use empty() for them too?

Simply because ```sizeof()``` is an alias of ```count()```, so I just kept the same function call where possible to avoid accidental logic changes as far as possible.